packages/native/src/components/Layout/List/IconBoxList/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import List, { BaseListItemProps, BaseListProps } from "../List";
+import { IconBox } from "../../../Icon";
+
+export type IconBoxListItemProps = Omit<BaseListItemProps, "bullet"> & {
+  Icon: (props: { size?: number; color?: string }) => React.ReactElement;
+};
+
+export type IconBoxListProps = Omit<BaseListProps, "items"> & {
+  items: IconBoxListItemProps[];
+};
+
+export default function IconBoxList({
+  items,
+  ...props
+}: IconBoxListProps): React.ReactElement {
+  return (
+    <List
+      items={items.map((item) => ({
```

Comment:
items should be memoized since it is passed to FlatList which is a pure component

Reply:
Done